### DISCUSSION TITLE
Hungarian language leader

### DISCUSSION THREAD

**Discussion by tanker11** [OHF-Voice/intents#329] [Language Leaders]
Hi,

I would be happy to help with Hungarian translations, especially for the grammar and expressions.

R

**Comment by nagyrobi** [comment]
https://github.com/home-assistant/intents/discussions/405
Szia! Mik a konkrét javaslataid?

**Comment by tanker11** [comment]
Szia! Hirtelen felindulásból jelentkeztem, mert szívesen segítek, ha tudok,
és nagyon is érdekel, hogy lehetőleg az élő nyelvben is használatos
kifejezések honosodjanak meg. Nem igazán tudom viszont, hogy hol kell
kezdeni. Átfutottam, ami itt van:
https://github.com/home-assistant/intents/tree/main/sentences/hu , de nem
tudom, hogyan született, és mekkora a szabadsági foka a kifejezéseknek.
Ha tudsz segíteni, hogy miben tudnék segíteni, akkor szívesen venném. :)
Belekotyogni nem akarok, csak ha tényleg elkél a segítség.


H. Árkosi Róbert ***@***.***> ezt írta (időpont: 2023. jan.
5., Cs, 14:06):

**Comment by nagyrobi** [comment]
Jó, egyelőre elvagyunk, köszi. Egyelőre még a koncepció és a műküödési elv is kidolgozás alatt van. Angolból kiindulva kezdték el fejleszteni a motort, nos, nem a magyar fonetika a legnagyobb problémájuk, ahogy látom...

Egyelőre az a cél, hogy pár alap-mondatot ismerjen fel a rendszer, lássuk, elvi szinten jó irányban haladunk-e. Utána jöhet a bővítés. Most még nem érdemes túlbuzogni mindenféle mondat kombinációval mert esélyes, hogy mindent újra kell majd írni...

Itt a GitHub-on jelezni fogom nektek, amikor kellene segítség.

**Reply by nagyrobi** [reply]
Addig is, ötletek ide jöhetnek: https://github.com/home-assistant/intents/discussions/368

**Comment by nagyrobi** [comment]
@vadaszi

**Comment by synesthesiam** [comment]
@nagyrobi Would you or others be able to translate the new responses into Hungarian?

**Reply by nagyrobi** [reply]
sure, I'll check it out tomorrow

**Reply by synesthesiam** [reply]
Thanks!

**Reply by nagyrobi** [reply]
Done!

**Comment by synesthesiam** [comment]
@nagyrobi when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Comment by ercsey** [comment]
Szia. Még mindíg nem érkezett el az ideje, hogy bővítsük kicsit a magyar mondatokat?

**Reply by nagyrobi** [reply]
Szerintem nyugodtan neki lehet látni! Köszi az érdeklődést!
